metricbeat/module/mongodb/status/conversion.go

```diff
@@ -0,0 +1,97 @@
+package status
```

Comment:
Yeah, I'm trying to find a way to organize these in a way that makes sense. We basically have two sets of convertors. One set that work on `map[string]string` (one level deep) and one that work on `map[string]interface{}` (nested). I'm thinking to have something like:

```
helper/
  schema.go
  mapstrstr/
    ...
  mapstriface/
    ...
```

The names are not great, I know. We could also either move the whole thing one level down under `helper/schema/` or replace `helper` with `schema` as there are no other helpers.